### DISCUSSION TITLE
Allow hiding the main bar in the BarHitErrorMeter

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#21669] [Ideas] [answered]
This : 

https://user-images.githubusercontent.com/74463310/208138897-2f931d08-e80f-490e-b30a-f12c90595ce7.mp4

as for why it would be desirable, some mania esque game players like having their hit error bars on their playfield ( as i believe is the default in https://github.com/etternagame/etterna and possibly some others) this option reduces the perceived clutter making for a better experience.

ps: is making a discussion for these small QoL changes before PRing preferable or should i just open the PRs directly

**Comment by peppy** [comment]
You can add a setting for this.

**Comment by mk56-spn** [comment, marked as answer]
closed by: https://github.com/ppy/osu/pull/22103
